pom.xml

```diff
@@ -12,8 +12,8 @@
 
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
-		<artifactId>spring-cloud-build</artifactId>
-		<version>2.2.0.RELEASE</version>
+		<artifactId>spring-cloud-dataflow-build</artifactId>
```

Comment:
It looks weird to have the parent artifact Id being `spring-cloud-dataflow-build` for the deployers. Can we come up with any other name that doesn't have `dataflow` in it? Perhaps, we can create `spring-cloud-deployer-build` that acts as a parent for all the deployers?

Reply:
There's going to exactly one parent and there's not good naming for dataflow/skipper/deployers other than they all belong to "dataflow" family.